### ISSUE TITLE
Taxonomy links don't work

### ISSUE THREAD

**Issue by modreoci** [solidusio/solidus#1010]
When I click any taxonomy link it appears the statement:
```
The page you were looking for doesn't exist.
You may have mistyped the address or the page may have moved.
If you are the application owner check the logs for more information.
```
log:
```
2016-03-28T12:42:04.163412+00:00 app[web.1]: Started GET "/t/kategorie/oblecenie" for 178.143.155.154 at 2016-03-28 12:42:04 +0000
2016-03-28T12:42:04.173057+00:00 app[web.1]:   Parameters: {"id"=>"kategorie/oblecenie"}
2016-03-28T12:42:04.189970+00:00 app[web.1]:   Spree::Taxon Load (1.6ms)  SELECT  "spree_taxons".* FROM "spree_taxons" INNER JOIN "spree_taxon_translations" ON "spree_taxon_translations"."spree_taxon_id" = "spree_taxons"."id" WHERE "spree_taxon_translations"."permalink" = '--- !ruby/object:ActiveRecord::StatementCache::Substitute {}
2016-03-28T12:42:04.218831+00:00 app[web.1]:   Rendered public/404.html (0.5ms)
2016-03-28T12:42:04.173028+00:00 app[web.1]: Processing by Spree::TaxonsController#show as HTML
2016-03-28T12:42:04.219039+00:00 app[web.1]: Completed 404 Not Found in 46ms (Views: 28.5ms | ActiveRecord: 4.7ms)
2016-03-28T12:42:04.189988+00:00 app[web.1]: ' AND "spree_taxon_translations"."locale" IN ('sk', 'en') LIMIT 1
2016-03-28T12:42:04.217174+00:00 heroku[router]: at=info method=GET path="/t/kategorie/oblecenie" host=bwshop.herokuapp.com request_id=b8dd214c-0277-45ba-bf43-fd6942d0729a fwd="178.143.155.154" dyno=web.1 connect=1ms service=58ms status=404 bytes=2216
2016-03-28T12:42:04.525428+00:00 heroku[router]: at=info method=GET path="/favicon.ico" host=bwshop.herokuapp.com request_id=a9fbe5f7-bde1-49f9-a566-f20349eb078b fwd="178.143.155.154" dyno=web.1 connect=1ms service=2ms status=200 bytes=143
```

**Comment by jhawthorn** [comment]
This doesn't seem to be an issue in solidus itself. Judging by that SQL query, maybe something to do with `solidus_globalize`. I'll investigate further.

**Comment by jhawthorn** [comment]
@modreoci I believe you're experiencing globalize/globalize#423. Using the current globalize master should fix this issue.

If it doesn't, please file a bug against [solidus_globalize](https://github.com/solidusio-contrib/solidus_globalize) and we'll take another look.

**Comment by modreoci** [comment]
Sorry I'm not very skilled in RoR. I tried it and it produce error in resolving dependencies:
```
Fetching git://github.com/globalize/globalize.git
Fetching gem metadata from https://rubygems.org/...........
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "globalize":
  In Gemfile:
    globalize

    solidus_globalize was resolved to 3.1.0.beta, which depends on
      globalize (~> 5.0.1)

```
Gemfile:
```ruby
source 'https://rubygems.org'

# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
gem 'rails', '4.2.5'

gem 'solidus', '1.1.1'
gem 'solidus_auth_devise'
gem 'rails-i18n'
gem 'devise-i18n'
gem 'globalize', github: 'globalize/globalize', branch: 'master'
gem 'solidus_i18n', github: 'solidusio-contrib/solidus_i18n', branch: 'master'
gem 'solidus_globalize', github: 'solidusio-contrib/solidus_globalize', branch: 'master'

# Use sqlite3 as the database for Active Record
gem 'pg'
# Use SCSS for stylesheets
gem 'sass-rails', '~> 5.0'
# Use Uglifier as compressor for JavaScript assets
gem 'uglifier', '>= 1.3.0'
# Use CoffeeScript for .coffee assets and views
gem 'coffee-rails', '~> 4.1.0'
# See https://github.com/rails/execjs#readme for more supported runtimes
# gem 'therubyracer', platforms: :ruby

# Use jquery as the JavaScript library
gem 'jquery-rails', '~> 3.1.4'
# Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
gem 'turbolinks'
# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
gem 'jbuilder', '~> 2.0'
# bundle exec rake doc:rails generates the API under doc/api.
gem 'sdoc', '~> 0.4.0', group: :doc
# for heroku
group :production do
  gem 'rails_12factor'
  gem 'puma'
end

# Use ActiveModel has_secure_password
# gem 'bcrypt', '~> 3.1.7'

# Use Unicorn as the app server
# gem 'unicorn'

# Use Capistrano for deployment
# gem 'capistrano-rails', group: :development

group :development, :test do
  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
  gem 'byebug'
end

group :development do
  # Access an IRB console on exception pages or by using <%= console %> in views
  gem 'web-console', '~> 2.0'

  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
  gem 'spring'
end
ruby "2.3.0"

```
